fe/fe-core/src/test/java/com/starrocks/sql/plan/ScanTest.java

```diff
@@ -437,7 +437,7 @@ public void testMetaScanWithCount() throws Exception {
                 "args nullable: false; result nullable: true], " +
                 "max[([max_t1a, VARCHAR, false]); args: VARCHAR; result: VARCHAR; " +
                 "args nullable: false; result nullable: true], " +
-                "dict_merge[([dict_merge_t1a, VARCHAR, false]); args: INVALID_TYPE; " +
+                "dict_merge[([dict_merge_t1a, ARRAY<VARCHAR>, false]); args: INVALID_TYPE; " +
```

Comment:
why intermediate result type changed? Is this safe? Will there be any compatibility issues during grayscale upgrade?

Reply:
good question, I modify here
```
                slotDescriptor.setType(entry.getKey().getType());
```

Reply:
I modify input type, not intermediate type. In actually, we use `array<varchar>` now, not `varchar`, I don't know why doesn't crash now....

Reply:
sorry, I misread, I thought this was the type of intermediate result before…

but no matter what, it seems that it is correct after your modification?